src/SIL.LCModel/DomainImpl/OverridesCellar.cs

```diff
@@ -2665,7 +2665,7 @@ public IFsFeatStrucType GetOrCreateFeatureTypeFromXml(XmlNode item)
 				// The XML is from a file shipped with FieldWorks. It is quite likely multiple users
 				// of a project could independently add the same items, so we create them with fixed guids
 				// so merge will recognize them as the same objects.
-				string guid = XmlUtils.GetAttributeValue(item.SelectSingleNode("fs"), "typeguid");
```

Comment:
GetAttributeValue would throw if there was no attribute present. Given there is no null check on the guid here this should really be a GetRequiredAttributeValue if such a method exists.